src/main/java/br/com/sbs/cubatech/category/SubCategory.java

```diff
@@ -2,25 +2,42 @@
 
 import br.com.sbs.cubatech.course.Course;
 
+import javax.persistence.*;
 import java.util.ArrayList;
 import java.util.List;
 
 import static br.com.sbs.cubatech.validation.Validator.*;
 
+@Entity
+@Table(name = "subCategory")
 public class SubCategory {
 
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
     private Long id;
     private String name;
     private String urlCode;
+
+    @Column(columnDefinition = "LONGTEXT")
     private String description;
```

Comment:
Creio que não é mais `LONGTEXT`.

```suggestion
    private String description;
```